client/my-sites/plans/current-plan/index.jsx

```diff
@@ -131,7 +123,11 @@ class CurrentPlan extends Component {
 				<QuerySitePlans siteId={ selectedSiteId } />
 				{ shouldQuerySiteDomains && <QuerySiteDomains siteId={ selectedSiteId } /> }
 
-				{ ! showThankYou && <PlansNavigation path={ path } /> }
+				<Dialog isVisible={ showThankYou }>
```

Comment:
I've sometimes noticed we show "free plan thank you" before the paid plan kicks in. Should we just add `!isRequestingSitePlans` check here? Totally not blocking and we can loop back to this later.

Reply:
Actually there already seems to be `this.isLoading` helper for that.